tests/testthat/_snaps/harmonic.md

```diff
@@ -73,7 +78,7 @@
     Condition
       Error in `step_harmonic()`:
       Caused by error in `prep()`:
-      ! `cycle_size` must be length 1 or the same  length as the input columns
+      ! `cycle_sizes` must be length 1 or the same length as the input columns.
```

Comment:
```suggestion
      ! `cycle_size` must be length 1 or the same length as the input columns.
```

I'm seeing that the argument name is the singular locally. Am I missing something here?

Reply:
good catch, it is always `cycle_size` but internally it is called `cycle_sizes` at times